DungeonMaster/Assets/Scripts/HomeUIManager.cs

```diff
@@ -112,18 +129,18 @@ void Start()
 
         yield return StartCoroutine(WaitForRequest(www, loginRoute));
         GoToMain();
-        StartCoroutine(PROFILE());
+        StartCoroutine(ProfileRequest());
     }
 
-    IEnumerator PROFILE() {
+    IEnumerator ProfileRequest() {
         UnityWebRequest www = UnityWebRequest.Get(URLPrefix + baseURL + "/" + profileRoute);
```

Comment:
Not a change on this pr but would suggest a function for building the url for these, as there will be more endpoints coming up.

Reply:
im working on figuring out a static class to handle all of the networking

Reply:
https://github.com/neuecc/UniRx

Reply:
gg.png ?

Reply:
https://github.com/oneleif/DungeonMaster-Client/issues/15